.github/workflows/ci.yml

```diff
@@ -734,3 +734,65 @@ jobs:
           find .
           cat external-result/*
           ! grep "fail" external-result/*
+
+  AUR:
+    name: Update AUR package
+    runs-on: ubuntu-latest
+    needs: [build]
+    if: github.event_name == 'release' && !github.event.release.prerelease
+    steps:
+      - name: Version number
+        id: version
+        shell: bash
+        run: |
+          VERSION="${{ github.event.release.tag_name }}"
+          echo "version=$VERSION" >> $GITHUB_ENV
+
+      - name: Ssh
+        env:
+          AUR_SSH_KEY: ${{ secrets.AUR_SSH_KEY }}
+        run: |
+          echo "$AUR_SSH_KEY" > ${{ github.workspace }}/aur_ssh_key
+          chmod 600 ${{ github.workspace }}/aur_ssh_key
+
+          mkdir -p $HOME/.ssh
+          echo "Host aur.archlinux.org" >> $HOME/.ssh/config
+          echo "  IdentityFile ${{ github.workspace }}/aur_ssh_key" >> $HOME/.ssh/config
+          echo "  User aur" >> $HOME/.ssh/config
+
+          ssh-keyscan -H aur.archlinux.org > $HOME/.ssh/known_hosts
+
+      - name: Fetch AUR package
+        run: |
+          git clone ssh://aur@aur.archlinux.org/toit.git
+
+      - name: Git
+        run: |
+          cd toit
+          git config user.email "leon@toit.io"
+          git config user.name "Leon Gungadin Mogensen"
+
+      - name: Update AUR package
```

Comment:
This is more involved than for the Jaguar build where we use `updpkgsums: true` and the archlinux-package-action. Add a comment to explain why that doesn't work?

Reply:
done.